test/networkd-test.py

```diff
@@ -176,15 +176,16 @@ def assert_link_states(self, **kwargs):
                 actual = fields[-1]
                 if (actual != expected and
                         not (expected == 'managed' and actual != 'unmanaged')):
-                    self.fail("Link {} expects state {}, found {}".format(iface, expected, actual))
+                    err_msg = "Link {} expects state {}, found {}".format(iface, expected, actual)
+                    self.fail(err_msg)
```

Comment:
Why this change?

Reply:
This was a hack to get it working during testing, before if you got an error, it would print `Link {} expects state {}, found {}` and not format. Trying to find a better solution